frontend/src/Services/api.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const url =
+  process.env.NODE_ENV === 'development'
+    ? process.env.REACT_APP_DEVELOPMENT_URL
+    : process.env.REACT_APP_PRODUCTION_URL;
+
```

Comment:
I think, we dont need to declare two URLs, just one name and for prod will be deployed different value for url